Styles.css

```diff
@@ -0,0 +1,107 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-size: 2rem;
+    color: #ffffff;
+    background-image: url("BG.jpeg");
+    /*https://images.unsplash.com/photo-1536559692556-79e8be88e8ab?w=2008*/
+    background-position: center;
+    background-attachment: fixed;
+    background-size: 100vw auto;
+    background-repeat: no-repeat;
+    font-family: "Helvetica", Arial, sans-serif;
+}
+
+@media only screen and (max-aspect-ratio: 4016/6016) {
+    body {
+        background-size: auto 100vh;
+    }
+}
+
+a {
+    text-decoration: inherit;
+    color: inherit;
+}
+
+header {
+
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    -ms-flex-align: flex-start;
+    align-items: flex-start;
+    padding: 20px;
+    height: 20vh;
+    min-height: 140px;
+}
+
+.Homebutton {
+    font-family: "Brush Script MT", Arial, sans-serif;
+    font-size: 3rem;
+    width: 7rem;
+}
+
+header a {
+    padding: 5px;
+}
+
+header nav {
+    /*display: -ms-flexbox;*/
+    display: flex;
+    -ms-flex-pack: center;
+    justify-content: center;
+}
+
+.topnav a {
+    margin: 0 10px;
+    flex-wrap: wrap;
+}
+
+.sociallinks {
```

Comment:
The clickable area of these icons is a bit small compared to every other element intended to be clicked on. I'd make them a little bigger.